algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -86,6 +85,11 @@ class TopPctCmsLaws extends PropSpec with PropertyChecks with Matchers {
   val SEED = 1
   val HEAVY_HITTERS_PCT = 0.1
 
+  implicit def topCmsEquiv[K]: Equiv[TopCMS[K]] =
```

Comment:
this is sketchy. What are your thoughts here? Tests fail without the equiv instance.